reference/requirements.md

```diff
@@ -79,15 +79,17 @@ Charmed Anbox Cloud requires LXD version >= 5.0.
 (sec-juju-version-requirements)=
 ### Juju
 
-The charmed Anbox Cloud requires [Juju 2.9 or later](https://juju.is/) to manage the different components and their dependencies. You can install Juju with the following command:
+The charmed Anbox Cloud requires a minimum of [Juju 3.0 or later](https://juju.is/) to manage the different components and their dependencies.
 
-    snap install --classic --channel=3.1/stable juju
+If you want to manage user secrets, use Juju 3.3 or later.
 
-If you wish to install a different version, replace `3.1` in the command with the desired version.
+We recommend using Juju 3.6, so that you will be using the latest version of Juju and can manage user secrets. See [Juju documentation](https://documentation.ubuntu.com/juju/latest/howto/manage-secrets/) for more information about managing secrets.
 
-To switch to the 2.9 series, use the following command:
+You can install Juju with the following command:
 
-    snap refresh --channel=2.9/stable juju
+    snap install --classic --channel=3.6/stable juju
```

Comment:
We should not add `--classic` to install juju from 3.6/ stable channel as it's now strit confinement

Reply:
Also, we're using channel `3/stable` in all other places. So for consistency, we may want to use 3/stable channel as well